vagrant-hosts.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |gem|
   gem.require_path = 'lib'
 
   # Pinned for compatibility with vagrant-spec.
-  gem.add_development_dependency 'rake', '~> 10.0'
+  gem.add_development_dependency 'rake', '~> 12.0'
```

Comment:
Have you confirmed that this does not cause a compatibility issue with `vagrant-spec`?